sdk/core/perf/src/arg_parser.cpp

```diff
@@ -93,6 +94,15 @@ Azure::Perf::GlobalTestOptions Azure::Perf::Program::ArgParser::Parse(
   {
     options.Warmup = parsedArgs["Warmup"];
   }
+  if (parsedArgs["TestProxies"])
+  {
+    std::string proxy;
+    std::istringstream fullArg(parsedArgs["TestProxies"].as<std::string>());
+    while (std::getline(fullArg, proxy, ';'))
```

Comment:
What is this loop doing? Maybe add a comment?

Reply:
Oh, I get it:
> More than one proxy address can be added using semicolon separated format.

Consider adding a comment right here as a nit, might be helpful for future maintenance.

Does this work if there is no semi-colon, and there's only one test proxy?

Reply:
> Does this work if there is no semi-colon, and there's only one test proxy?

Yes, `std::getline`  takes all from first arg to the first delimiter ocurrence OR the end of the buffer